pyfolio/bayesian.py

```diff
@@ -86,7 +86,7 @@ def model_returns_t_alpha_beta(data, bmark, samples=2000):
                  mu=mu_reg,
                  sd=sigma,
                  observed=y)
-        trace = pm.sample(samples)
+        trace = pm.sample(samples, progressbar=False)
```

Comment:
Seems like that fixed the travis issue. This isn't a proper fix, however. Can you make this a kwarg for the top function that defaults to `True` and only set it to `False` in the unittest?